cfgov/unprocessed/apps/analytics-gtm/js/enterprise-attribution-cookies.js

```diff
@@ -201,7 +201,7 @@ const sourceCookies = {
    */
   parseOrganicSearch: function( host, query, referrer ) {
     // Referrer must not be the same host
-    if ( hostsAreEqual( 'http://' + host, referrer ) === false ) {
+    if ( hostsAreEqual( 'https://' + host, referrer ) === false ) {
```

Comment:
Is this a file we should be modifying?

Reply:
Yes, `host` is set by `document.location.host` on https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/analytics-gtm/js/enterprise-attribution-cookies.js#L414 and https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/analytics-gtm/js/enterprise-attribution-cookies.js#L53, which would be our pages where this script runs, which will be `https`.